src/features/game/scenes/gameManager/GameGlobalAPI.ts

```diff
@@ -283,6 +284,12 @@ class GameGlobalAPI {
     await this.getGameManager().getDialogueManager().showDialogue(dialogueId);
   }
 
+  public async showNextLine() {
+    await this.getGameManager()
+      .getDialogueManager()
+      .showNextLine(() => {});
```

Comment:
Is `() => {}` necessary here?

Reply:
The showNextLine in GameDialogueManager takes in a parameter of void function.

Reply:
Right, and going one step further, my suggestion would be to keep the signature consistent for `showNextLine` even in GameGlobalAPI, i.e. 

```ts
public async showNextLine(resolve: () => void) { ... }
```

This will also require some changes for usages of `showNextLine`.

Reply:
Comment addressed:  commit [3ba899190aa5f5c00d10b9c6a950f3dc6682a1df](url)